tutorials/notebooks/mct_features_notebooks/keras/example_keras_xquant.ipynb

```diff
@@ -0,0 +1,208 @@
Zb0YH97sYrRG3DAOFwSbHt2L7Z9TfCZkYMG2B9ejWcS7UQQpyOEIaBYYixQyLJMIwDMNAY9CddIQUwmglP4B1BhASgtT3R4meqmUHHQA9Dub0Q6Zc99fXgOYSRG4UPKWGfqHAJjoFByIu/MbCZQUd0iRNESGKKyIN3CkYRxnB1IFYk9bJhG9qA8kS1M5/7c/r6gW9Drn+U0cVntBXtwDFoUwx1wAgn/0bw5fsQZfGfUbYwtBXth8cFxjtCBwY/XhDhBLLZQrnIe0EOEPpfFJwTfLuF4XvKQhlpN9oA4YH24BrSFuD7HteYfhBsC17+2LQL/0Ys4ljZ/u44r68Lf4fPw3E4brAcwetM+9Lf8WTJJclxTfcXVakrDxGPRBEhcoSHXf++PW3cmOGpv5unbtCltm3rRjtYuM8t1StEbXHyZGqgdKjItm7Z4DxFWrZsYQP797bRI75ZFk8IIUT9gNFHjgWMqmDySw8u8xh+CAuZZu+DYJS+/PLLzpBiFpyZdm+E1iZ4iCBcxK0WwionGJB4g3hPE4xMb9R5r4YgGJaIEYBx7o3fODBCmZWnjfAAoUxBOB4JUvGawKDEQySJIIIBTO4SDGbazycpjQPDmbJiDGNMh8FA5jvwoR9AmSkPYUDUIQyfUQYvAvEiN8f9998f+Zo5c6bbj7Cb++67z31WXeJRykvoD/0G4SdYvlzgmuKRgxcLfTAKLxhwrijhKwx5NPDOYPugOIXgF24D/6L+PtSJduGzu+++u1IQyuZ3xz7sGz6Hf33nO9+pPK6/PpTNwzWkXbjWSeobpqb7i/pFoogQNYCEq2NGDXGvCmGkzLZsXu/CG8g1gjHLw12I7GH2goz/5bZ753bbsG61HS455AQRxLhJZ5+RepjXTYI1IYQQ8WDk4M7P833p0qXOSPNgBOHyz2z6kCFDqhirfId3BMZ0cGzA58zSE5rAsrF4ONSFIAKIFn61EEIVECg8GMOEJ2DAhQUFwkQQKCgjoQ/B8rMPHh0YmJk8M9hn3759LlkoSUcxXC+77LIqdWUbnyCV8912223uvToQB0jwiTBC7gif7DQThHxgZCMOcR2DbeHLwbXCaEYI8bAf1997V/Du4fwsTYy3A6vaJBFzsoEyElbCssEIVJSrNjyK8PAYPHiwKzcJUIN1AkQIBBP6BqFPmUAMYOlmwsDo27WVKDjX350QSWh24MABWWxC1JCSw2X2+Yp1tvrLzakbc4WXSJu27axnrz7WuUt3a9sm9VBMPbAqnlk1e3CJpkzqduz+97UdP3bMDh4stAOF+1zYDAOeNm1a2aD+fWzKpDHWpZMe+KLp4A0s3hn0884gl5lJBucYIULkE/RLwjQQCTDWMMQw7vGYwDDG4A8b9BhyGNFsf/vtt7s8C4BXBEYk93lmuTMJIhik5CaIgvwMs2fPdiEG99xzjzPmo8DA9auI8NsiNwW/NYxt3uOEGfIuIDzwGyUPBx4e5H/gvPxm48QI6o2QQhiED43ASMYTIBz2gQhBHSgb7eRDeKKgfoTVYNDjIYIwQ5lpw0wiATk5fPJXBB3qxP2Ga0UoD8fYs2eP8zzgb5ZD9tt7uMZz5sxx9ef8iBPUzbch/04a8oM3BQICHhIs6+s9dIKQWwURhDb04g3nIMFpEgEgyTmCdeJ7Esnyzjm3bNnizkvfYEWVcN+gvcjjglDhBRW8cEhgS9LapKINbTd37lzXR+kfYY+ZXH53cVT3e0nSZpmo6f5NnSS/jZri+2ISJIoIUUuUlx+19Zu22xerN9ihktLUw7HiocXNum279u7VqiUPd4kiIg4MweOpm3i5HSn/ZvCIF1LHgg42ZtRgO+uMoakHqzxERNNCoohojJCHgBwMhAl4QxXjD7Hg+uuvP6Xf4iVCskCM6DvuuKPScHv//fftvffec39XB54YN954Y/pfVUkqigBGLoZ2MPwDLwAMWEQAjhFF1H7UZ/r06S7sJ2wsg68fRhBeKiTBxIsialtfB3771UH9fD3nz5/vBI4kkKQ0GCaB9wqhSxj2/l4ECDeXX365M7yjjPqo608bIhzQhkmN4CTGs68fIT/k50Ck4T2qDaNIaqBzv8XTBQHMj0Ggumvsr5sXl9iOBLqUNxuqE0Ug299dHBJFGhaJIkI0cYqKD9uK1Rts05adduTYcTt54qSdSA/yhUgKQggDj/bt2rhwmTGjhiqpqmiySBQRjRn6KiEDgIGVyfjxiSCTGrN1Db8vQg4wKqvzUgni90MEwPBN6gmQz2RzHYP4/bJtw3yG+7BPcJqv1zjX65UUL+ax8s8NN9yQdf1run9TR6KIEKcJh0vLbfPWXbZz9z4rPFBs5UdPTeIlkuONJQYbTf3B0ib1cO/etZP17d3dJVTtlvpbiKaMRBEhhBD5AmFAs2bNciE5eJ9kypUTRU33Px2QKCLEaQiD/KPHjrtXarSf/lQkBQOJRGi4U+LCSRx2U3NDpFcg9bRu3Sr1oGhjzTWjIE4jJIoIIYTIB8ipsnDhwpwTH9d0/9MFiSJCCJElGEnELxPLDAgjxNIqu7gQTQOJIkIIIRoanj2vvPKKW3mIpMFTp07NStCo6f6nExJFhBAiS0j2xVKBqO4ebqZkiSfTuBCicSNRRAghRD7AswfI/ZMLNd3/dEGiiBBCZAkPGJbGI3wmCEv1kcmejONKYCVE40WiiBBCCCEaCvnzCCHyHgyk4NJwHjLfs4QeYknU90IIIYQQQgiRCYkiQoi8x88eR8Fs8r59+2zr1q1OJBFCCCGEEEKIpEgUEULkPZlEEeA7sn1v3rw5q/hBIYQQQgghxOmNRBEhRN6TSRRp0aKFW42mXbt2Vl5e7oSRkpKS9LdCCCGEEEIIEY9EESFEoyBOFCG7d48ePWzMmDE2btw4Gz58uFv+TDlGhBBCCCGEENUhUUQIkfd4TxFWmGHFmeAyXqxOUVRU5Lbx3+M5ggeJEEIIIYQQQsRj9v8B0OgcHYMxUskAAAAASUVORK5CYII=)"
```

Comment:
Also, why the link of this image is so long?

Reply:
I don't know, to be honest... I guess this is how it is stored when you add an image to a colab notebook...